client/components/paginated-survey/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+import QuestionStep from './components/question-step';
+import { usePaginatedSurveyContext } from './context';
+import { Question } from './types';
+
+type PaginatedSurveyType = {
+	questions: Question[];
+	recordTracksEvent: ( eventName: string, eventProperties: object ) => void;
+};
+
+const PaginatedSurvey = ( { questions, recordTracksEvent }: PaginatedSurveyType ) => {
+	const { currentPage, nextPage, previousPage } = usePaginatedSurveyContext();
+	const [ answers, setAnswers ] = useState( {} ); // State management TBD
+
+	const onChangeAnswer = ( questionKey: string, value: string[] ) => {
+		setAnswers( { ...answers, [ questionKey ]: value } );
+	};
+
+	const currentQuestion = questions[ currentPage - 1 ];
+
+	return (
+		<QuestionStep
+			previousPage={ previousPage }
+			nextPage={ nextPage }
+			skip={ nextPage }
```

Comment:
I wonder if we want to know if the questions have been skipped, but this is more on the storing the answers part (cc @JuanLucha).

Reply:
We want to know, but I'm not sure if we are going to find out by comparing the questions with their answers instead of specifically storing a "skipped" status. Something to think about, indeed.